c3pzero_base_config/config/base_config.yaml

```
@@ -0,0 +1,28 @@
+###############################################################
+#
+#  This configures the robot to work with moveit studio
+#
+###############################################################
+
+hardware:
+  # Used by the ur_description package to set kinematics and geometry for a specific robot type.
+  type: "c3pzero"
+  simulated: True
+
+  # The robot's IP address
+  ip: "!!! Please set the IP address of the robot"
+
+  # launch files
+  view_robot_base_launch_file:
```

The Studio Agent expects to find a key with a specific name in this file:

```suggestion
  robot_driver_persist_launch_file:
```